Formula/lavinmq.rb

```diff
@@ -19,6 +19,10 @@ def install
     bin.install "bin/lavinmqperf"
   end
 
+  service do
+    run [opt_bin/"lavinmq", "-D", "/opt/homebrew/var/lavinmq"]
```

Comment:
Probably want to make the `"/opt/homebrew/var/lavinmq"` more "platform agonstic", think this would work, but wasn't able to try it:

```suggestion
    run [opt_bin/"lavinmq", "-D", var/"lavinmq"]
```